docs/topics/email.txt

```diff
@@ -393,9 +397,9 @@ Django's email library, you can do this using the
 .. class:: EmailMultiAlternatives
 
     A subclass of :class:`~django.core.mail.EmailMessage` that has an
-    additional ``attach_alternative()`` method for including extra versions of
-    the message body in the email. All the other methods (including the class
-    initialization) are inherited directly from
+    additional ``attach_alternative()`` method and ``alternatives`` attribute
+    for including extra versions of the message body in the email. All the other
+    methods (including the class initialization) are inherited directly from
     :class:`~django.core.mail.EmailMessage`.
```

Comment:
```suggestion
    A subclass of :class:`~django.core.mail.EmailMessage` that allows
    additional versions of the message body in the email via the
    ``attach_alternative()`` method. This directly inherits all methods
    (including the class initialization) from
    :class:`~django.core.mail.EmailMessage`.
```
This is based off [this discussion](https://github.com/django/django/pull/18278#discussion_r1644179202).